tls/s2n_kem.h

```diff
@@ -28,17 +27,19 @@ typedef uint16_t kem_private_key_size;
 typedef uint16_t kem_shared_secret_size;
 typedef uint16_t kem_ciphertext_key_size;
 
+#define IN /* Indicates a necessary function input */
+#define OUT /* Indicates a function output */
```

Comment:
This is cool, but it's not related to the consolidated PQ check and should probably be in its own PR with better visibility. Maybe we want to do it everywhere?